tests/test_move_embargo_args.py

```diff
@@ -1,29 +1,40 @@
 import unittest
-import sys
-sys.path.insert(0,"..")
-from src import move_embargo_args
+# import sys
+# sys.path.insert(0, "..")
+# from src import move_embargo_args
+import move_embargo_args
+
 
 class TestEmbargoArgs(unittest.TestCase):
     def test_parser(self):
         """
         Test the parser without having to execute the code
         """
-        
-        
-        
-        args = ['-f', '/repo/embargo', '-t', '/home/j/jarugula/scratch', '--instrument', 'LATISS', '-days', '30', '--datasettype', 'raw', '--collections', 'LATISS/raw/all', '--band', 'g']
-        
-        parser = move_embargo_args.parse_args(args)
+        args = ['-f', '/repo/embargo',
+                '-t', '/home/j/jarugula/scratch',
+                '--instrument', 'LATISS',
+                '-days', '30',
+                '--datasettype', 'raw',
+                '--collections', 'LATISS/raw/all',
+                '--band', 'g']
+        parser = move_embargo_args.parser(args)
         self.assertTrue(parser.long)
-        
         '''
```

Comment:
If this is no longer relevant, you should maybe remove it. If it's to remind you of things to add in the future, it's OK, but adding a "TODO" in front would help indicate that to the reader.  And in that case comments are better than a text block (which otherwise might be mistaken for a docstring).

Reply:
@beckynevin This is a useful comment. Let me know if you would like to change this code.